client/webserver/locales/en-us.go

```diff
@@ -236,4 +236,9 @@ var EnUS = map[string]string{
 	"Update TLS Certificate":      "Update TLS Certificate",
 	"registered dexes":            "Registered Dexes:",
 	"successful_cert_update":      "Successfully updated certificate!",
+	"export_wallet":               "Export Wallet",
+	"pw_for_wallet_seed":          "Enter your app password to show the wallet seed. Make sure nobody else can see your screen. If anyone gets accecss to the wallet seed, they will be able to steal all of your funds.",
+	"export_wallet_disclaimer":    "You WILL LOSE FUNDS if export your wallet and use the external wallet while you have active trades running in the DEX. It is recommended that you do not export your wallet unless you are an experienced user and you know what are doing.",
```

Comment:
I don't know if it's worth mentioning, but copying anything will put that plaintext into your clipboard which any program can easily access. That's why words copied with pencil and paper are potentially better. But, I think it's fine to let advanced users decide for themselves if they want to take the risk.

Reply:
How much more risky is copying to the clipboard than just typing the password? Is accessing someone's clipboard that much easier than installing a keylogger?

Reply:
It seems clipboard peeking is far worse,

I got curious and tried to find some info on it, but didn't seems to find good examples right away, however I tired the following:
- I've installed [a simple go binary that works with clipboard](https://github.com/golang-design/clipboard) (no root permissions required for this I think)
- then I just ran `gclip --paste` (no root needed as well) and basically got clipboard contents

^ I think this means pretty much any binary on my system (Mac, but I think this is applicable to most OS) running in user space can peek secrets in clipboard at any time